Error: Illegal state: service accessor is only valid during the invocation of its target method

```
notificationsAlerts.js:31 Error: Illegal state: service accessor is only valid during the invocation of its target method
    at illegalState (errors.js:155:16)
    at Object.get (instantiationService.js:75:31)
    at NewEditSessionAction.run (chatClearActions.js:188:47)
    at async ActionRunner.runAction (actions.js:112:9)
    at async ActionRunner.run (actions.js:104:13)
    at async MenuEntryActionViewItem.onClick (menuEntryActionViewItem.js:157:13)
onDidChangeNotification @ notificationsAlerts.js:31
(anonymous) @ notificationsAlerts.js:22
_deliver @ event.js:966
_deliverQueue @ event.js:977
fire @ event.js:1000
addNotification @ notifications.js:123
error @ notificationService.js:170
(anonymous) @ compositePart.js:181
_deliver @ event.js:966
_deliverQueue @ event.js:977
fire @ event.js:1000
run @ actions.js:109
await in run
onClick @ menuEntryActionViewItem.js:157
(anonymous) @ actionViewItems.js:106
```

--- a/src/vs/workbench/contrib/chat/browser/actions/chatClearActions.ts
+++ b/src/vs/workbench/contrib/chat/browser/actions/chatClearActions.ts
@@ -186,6 +186,7 @@ export function registerNewChatActions() {
 			const widgetService = accessor.get(IChatWidgetService);
 			const chatEditingService = accessor.get(IChatEditingService);
 			const dialogService = accessor.get(IDialogService);
+			const viewsService = accessor.get(IViewsService);
 			if (!(await this._handleCurrentEditingSession(chatEditingService, dialogService))) {
 				return;
 			}
@@ -201,7 +202,6 @@ export function registerNewChatActions() {
 				}
 			} else {
 				// Is running from f1 or keybinding
-				const viewsService = accessor.get(IViewsService);
 				const chatView = await viewsService.openView(EDITS_VIEW_ID) as ChatViewPane;
 				const widget = chatView.widget;
 

